expirer/src/main/scala/com/gu/media/expirer/ExpirerLambda.scala

```diff
@@ -19,26 +19,34 @@ class ExpirerLambda extends RequestHandler[Unit, Unit]
   with Logging
   with CapiAccess
   with YouTubeAccess
-  with YouTubeVideos {
+  with YouTubeVideos
+  with YouTubeClaims {
 
 
   override def handleRequest(input: Unit, context: Context): Unit = {
     val now = Instant.now()
     val oneDayAgo = Instant.now().minus(1, ChronoUnit.DAYS)
-    val assets = getVideosFromExpiredAtoms(1, 100, oneDayAgo, now, Set.empty).filter(isManagedVideo)
+    val atomsWithAssets: Seq[AssetDetails] = getVideosFromExpiredAtoms(1, 100, oneDayAgo, now, Seq.empty)
```

Comment:
out of scope for this PR: this lambda should really hit and endpoint in MAM rather than do this logic itself